test/llvm.fshl.ll

```diff
@@ -0,0 +1,63 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
One more comment regarding filename in spite of recently merged #780: this can be just named `fshl.ll`, but should be placed within `llvm-intrinsics` subfolder

Reply:
Sure, I've rebased & moved the file.